components/data-table/index.jsx

```diff
@@ -342,6 +359,19 @@ class DataTable extends React.Component {
 		}
 	};
 
+	onScrollerScroll = () => {
+		if (this.props.hasMore && this.props.onLoadMore) {
+			const scroller = this.scrollerRef;
+			if (
+				scroller &&
+				scroller.scrollTop + scroller.offsetHeight >
+					scroller.scrollHeight - this.props.loadMoreOffset
+			) {
+				this.props.onLoadMore();
```

Comment:
It's possible that this callback is invoked multiple times as the user scrolls within the `loadMoreOffset` area.  Is that ok?  The caller may not expect to have this invoked repeatedly.

Would it make more sense to avoid repeatedly calling it until the next time the table height changes?

Reply:
Yes currently it would be called many times if the user keeps scrolling within the `loadMoreOffset`. I was considering an additional flag for the consumer to pass in to indicate it is currently loading but that flag would not affect any rendering of the component so thought it wasn't appropriate.  I thought we would just leave it to the consumer to handle multiple calls.

If the only call this callback the first time until the table height changes, what if the height doesn't change and they still want to load more (i.e. like in an error case that failed to load more).

Reply:
What do you think about invoking it only when it crossing the `loadMoreOffset` threshold?  We'd have to keep track of the previous scroll position.

Reply:
Sure I could do that... then it would only be called if they scrolled up then scrolled back down... is that any better?

Reply:
I think it's _slightly_ better.  You're right that multiple calls are still possible, and that we should definitely mention that in the comments for the prop.

Reply:
I've update the comment to be:

This function fires when infinite loading loads more data.

This will be called multiple times while the table is being scrolled within the `loadMoreOffset`.  Please track whether or not loading is in progress and check it at the start of this function to avoid executing your callback too many times.